src/blog/2018-12-24/anouncing-stryker-0-34.pug

```diff
@@ -0,0 +1,128 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Install Stryker via the command line:
+        
+        ```shell
+        npm i stryker stryker-api --save-dev
+        ```
+
+        Or run `stryker init` in the root of your project:
```

Comment:
maybe `root folder`? But honestly dunno

Reply:
I don't think that makes it more clear. I'll leave it like it is.